Add CRON to build infra to catch dependency breakages

`xmlschema` released a new version (1.5.0) on Friday (5 Feb 2021) that moved around some of the objects / functions in the library.

```python
c:\users\dmt\anaconda3\envs\aicsimageio\lib\site-packages\ome_types\__init__.py:18: in <module>
    from .schema import to_dict, to_xml, validate  # isort:skip
c:\users\dmt\anaconda3\envs\aicsimageio\lib\site-packages\ome_types\schema.py:13: in <module>
    from xmlschema.converters import ElementData, XMLSchemaConverter
E   ImportError: cannot import name 'ElementData' from 'xmlschema.converters' (c:\users\dmt\anaconda3\envs\aicsimageio\lib\site-packages\xmlschema\converters\__init__.py)
```

This is just to alert that any sort of build systems that install `ome-types` (and `xmlschema` as a dependency) will likely fail today :)

@tlambert03 I will make a PR to resolve this later today probably. Alright if I also in that PR include a change to the `build` GitHub workflow to add a CRON job for this?

We have CRON jobs on `aicsimageio` build for these types of dependency catching: [example](https://github.com/AllenCellModeling/aicsimageio/blob/main/.github/workflows/build-main.yml#L8)

--- a/.github/scripts/download_aics_test_data.py
+++ b/.github/scripts/download_aics_test_data.py
@@ -6,9 +6,17 @@ from quilt3 import Package
 def download_test_resources() -> None:
     root = Path(__file__).parent.parent.parent
     resources = (root / "aicsimageio" / "aicsimageio" / "tests" / "resources").resolve()
+
+    # Get the specific hash for test resources
+    with open(root / "aicsimageio" / "scripts" / "TEST_RESOURCES_HASH.txt", "r") as f:
+        top_hash = f.readline()
+
+    # Download test resources
     resources.mkdir(exist_ok=True)
     package = Package.browse(
-        "aicsimageio/test_resources", "s3://aics-modeling-packages-test-resources"
+        "aicsimageio/test_resources",
+        "s3://aics-modeling-packages-test-resources",
+        top_hash=top_hash,
     )
     package["resources"].fetch(resources)
 

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -4,6 +4,11 @@ on:
   push:
     branches:
       - "master"
+  schedule:
+    # <minute [0,59]> <hour [0,23]> <day of the month [1,31]> <month of the year [1,12]> <day of the week [0,6]>
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+    # Run every Monday at 18:00:00 UTC (Monday at 10:00:00 PST)
+    - cron: '0 18 * * 1'
 
 jobs:
   build:
@@ -22,6 +27,7 @@ jobs:
           rm -rf build
           rm ome_types/_version.py
       - name: Commit
+        if: github.event_name == 'push'
         uses: EndBug/add-and-commit@v7
         with:
           add: "ome_types"

